### PR TITLE
Modernize: How to Detect and Count Objects in Zone

### PR DESCRIPTION
**Pull request by LinasKo**
# Description

Updated notebook: "How to Detect and Count Objects in Zone" to use latest `supervision` API (0.20.0)

## Type of change

Please delete options that are not relevant.

-   [x] Bug fix (non-breaking change which fixes an issue)

## How has this change been tested, please provide a testcase or example of how you tested the change?

Ran locally, but did NOT check if Colab was built successfully.

## Any specific deployment considerations

I did not have `nvcc`, so one field returns a "nvcc not found" response.

Was this originally run in Colab and then imported into the repo?

## Docs

-   [ ] Docs updated? What were the changes:

### CODE REVIEW

**Review by SkalskiP — APPROVED** [review]

**Comment by SkalskiP** [comment]
Hi @LinasKo! 👋🏻 Awesome work! I proactively approved it, but it would be great if you could fix those two things:

- Make the color of the zone the same as the color of the boxes.

Screenshot 2024-06-13 at 14 31 21
Screenshot 2024-06-13 at 14 31 10

- Fix those deprecation warnings.

Screenshot 2024-06-13 at 14 31 37
Screenshot 2024-06-13 at 14 31 29

**Comment by LinasKo** [comment]
Will do, thanks!

**Comment by LinasKo** [comment]
Addressed. @SkalskiP, could you merge this, please? I don't have write access, it seems.

**Pull request merged**
